test-series/1/test-1.1.0.ipynb

```diff
@@ -0,0 +1,1263 @@
+{
```

Comment:
<p>It would be helpful to see the fg line at high k_par. It's unclear if this shows sum = fg + eor.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera-validation/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
I think there was some discussion on the telecon about whether or not to extend the y-range and, but just for completeness, this is what it looks like to extend it (not boosted and boosted, respectively):
 ![image](https://user-images.githubusercontent.com/25778615/78942377-74099e00-7a6e-11ea-8a75-b7fde39e025d.png)
![image](https://user-images.githubusercontent.com/25778615/78942409-8a175e80-7a6e-11ea-8bba-2fe8680bce43.png)

To capture any of the foreground residuals at high k_par for the highest frequencies, even for the non-boosted spectra, the y-axis needs to go to -1.

Reply:
@aelanman which do you prefer?

Reply:
I agree that the "boosted" plot does kinda obscure the deviation, which is the point of the plot. However, I think the discrepancy is only interesting if it can't be explained by the remaining foreground power, and it's unclear from either plot if that's the case.

Reply:
<p>OK. I've added in a P_FG + P_EOR curve into each residual plot, to help show that <code>pspec</code>  is not doing anything strange -- it's treating the visibilities as uncorrelated.</p>

Reply:
Thanks! That looks good to me.